src/cr/cube/stripe/measure.py

```diff
@@ -198,6 +198,39 @@ def scale_mean(self):
         numeric value does not contribute to either the numerator or denominator of that
         computation.
         """
+        # --- value is None when no row-element has been assigned a numeric value ---
+        if not self._numeric_values.size:
```

Comment:
I'd like to be explicit here. I know that this works when size is 0 but i do prefer to have an explicit expression
e.g. `if self._numeric_values.size == 0 `

Reply:
yeah, ok. will do